pygmt/datasets/tile_map.py

```diff
@@ -79,6 +88,14 @@ def load_tile_map(region, zoom="auto", source=None, lonlat=True, wait=0, max_ret
         will stop trying to fetch more tiles from a rate-limited API [Default
         is ``2``].
 
+    zoom_adjust : int or None
+        Optional. The amount to adjust a chosen zoom level if it is chosen
+        automatically. Values outside of -1 to 1 are not recommended as they
+        can lead to slow execution. [Default is ``None``].
+
+        .. versionadded:: 0.11.0
```

Comment:
Maybe we should not add this until PyGMT reaches v1.0.0?
```
.. versionadded:: 0.11.0
```

Reply:
Ok, changed to using the [`note`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-note) directive instead at 0b016645f6f778873168f904c1a1ba803c3210a6. I've also used the `note` and `important` directive/admonition in other parts of the docstring at 32c04f8192e1b18206861f5839783ab8d6e9d500:

![image](https://github.com/GenericMappingTools/pygmt/assets/23487320/81db6104-ee1a-4a64-9608-587f2d4e6c03)

If it looks a bit messy, I can also revert the changes back to the plain text version.